src/map/lua/lua_baseentity.cpp

```diff
@@ -13551,6 +13551,57 @@ void CLuaBaseEntity::useMobAbility(sol::variadic_args va)
     // clang-format on
 }
 
+/************************************************************************
+ *  Function: triggerDrawIn()
+ *  Purpose : Forces a mob to use DrawIn on the mob's current target
+ *  Example : mob:triggerDrawIn(bool includeParty, CLuaBaseEntity* PEntity)
+ *  Note    : Params can assume a default value by passing nil
+ *          : e.g. triggerDrawIn(true) to pull in a party/alliance
+ ************************************************************************/
+inline int32 CLuaBaseEntity::triggerDrawIn(CLuaBaseEntity* PMobEntity, sol::object const& includePt, sol::object const& drawRange, sol::object const& maxReach, sol::object const& target)
```

Comment:
No reason to inline any of these functions, as the compiler is smart enough.  Also, given that there is only one retval, this should be a void.